README.md

```diff
@@ -238,16 +238,25 @@ To report security vulnerabilities or for questions about security, please conta
 Help improve Public Lab software!
 
 * Join the plots-dev@googlegroups.com discussion list to get involved
+* Most of the contributors network on our [gitter chat](https://gitter.im/publiclab/publiclab)
 * Look for open issues at https://github.com/publiclab/plots2/labels/help-wanted
 * We're specifically asking for help with issues labelled with [help-wanted](https://github.com/publiclab/plots2/labels/help-wanted) tag
 * Find lots of info on contributing at http://publiclab.org/wiki/developers
+<<<<<<< HEAD
```

Comment:
Plz, resolve the conflict!

Reply:
Hope it solved now, i deleted wrong commits